### PR TITLE
sync zh-trans  content/zh/docs/concepts/workloads/pods/ephemeral-containers.md

### PR DESCRIPTION
**Pull request by tanjunchen**
sync zh-trans  content/zh/docs/concepts/workloads/pods/ephemeral-containers.md

### CODE REVIEW

File: content/zh/docs/concepts/workloads/pods/ephemeral-containers.md [hunk 1]
```diff
@@ -164,11 +162,11 @@ enabled and kubernetes client and server version v1.16 or later.
...
 <!--
 The examples in this section demonstrate how ephemeral containers appear in
-the API. Users would normally use a `kubectl` plugin for troubleshooting that
-would automate these steps.
+the API. You would normally use a `kubectl` plugin for troubleshooting that
+automates these steps.
 -->
 
-本节中的示例演示了临时容器如何在 API 中出现。用户通常会使用 `kubectl` 插件来进行故障排查，从而自动化执行这些步骤。
+本节中的示例演示了临时容器如何出现在 API 中。 通常，您可以使用 `kubectl` 插件进行故障排除，从而自动化执行这些步骤。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/ephemeral-containers.md]: troubleshooting 译为故障排查更合适

File: content/zh/docs/concepts/workloads/pods/ephemeral-containers.md [hunk 2]
```diff
@@ -287,13 +285,26 @@ kubectl attach -it example-pod -c debugger
 ```
 
 <!--
-If process namespace sharing is enabled, you can see processes from all the containers in that Pod. For example:
+If process namespace sharing is enabled, you can see processes from all the containers in that Pod.
+For example, after attaching, you run `ps` in the debugger container:
 -->
 
-如果启用了进程命名空间共享，则可以从该 Pod 的所有容器中查看进程。如下所示：
+如果启用了进程命名空间共享，则可以从该 Pod 的所有容器中查看进程。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/ephemeral-containers.md]:
```suggestion
如果启用了进程命名空间共享，则可以查看该 Pod 所有容器中的进程。
```

**Comment by tanjunchen** [comment]
/cc @tengqm
done

**Pull request merged**